src/main/java/domain/card/Cards.java

```diff
@@ -0,0 +1,77 @@
+package domain.card;
+
+import domain.CardCalculator;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cards {
+    private static final String DELIMITER = ", ";
+    private static final int DEALER_STANDARD_ADDITIONAL_CARD = 16;
+    private static final String NO_CARDS = "입력한 카드가 없습니다.";
+    private static final int BLACK_JACK_SIZE = 2;
+    private static final int STANDARD_ACE_ELEVEN = 21;
+
+    private List<Card> cards;
+
+    private Cards(List<Card> cards) {
+        this.cards = cards;
+    }
+
+    public static Cards of(Card... cards) {
+        if (cards == null || cards.length == 0) {
+            throw new NullPointerException(NO_CARDS);
+        }
+
+        List<Card> cardList = new ArrayList<>();
+        Collections.addAll(cardList, cards);
+        return new Cards(cardList);
+    }
+
+    private void validateNullCards() {
+        if (this.cards == null || this.cards.isEmpty()) {
+            throw new NullPointerException(NO_CARDS);
+        }
+    }
+
+    public boolean isCardsSumUnderSixteen() {
+        validateNullCards();
+        return CardCalculator.calculateAceStrategy(this) <= DEALER_STANDARD_ADDITIONAL_CARD;
+    }
+
+    public boolean containAce() {
+        validateNullCards();
+        return this.cards.stream()
+                .anyMatch(Card::isAce);
+    }
+
+    public void addCard(Card card) {
+        if (card == null) {
+            throw new NullPointerException(NO_CARDS);
+        }
+
+        this.cards.add(card);
+    }
+
+    public int getCardsSum() {
+        return CardCalculator.calculateAceStrategy(this);
+    }
+
+    public boolean isBlackJack() {
+        return containAce() && this.cards.size() == BLACK_JACK_SIZE
```

Comment:
```suggestion
        return containAce()
                && this.cards.size() == BLACK_JACK_SIZE
```
조건에 따른 일관된 엔터는 읽기 좋은 코드를 만듭니다.